src/utils/hooks/useSplitTime.ts

```diff
@@ -19,7 +19,7 @@ const useSplitTime = (
       }
 
       if (timeLeft === 0 && periodLength) {
-        setTimeLeft(periodLength);
+        setTimeLeft(periodLength / 1000);
```

Comment:
Why do we divide by 1000 here, the period length is already in ms ?

Reply:
is it? So something else need to be wrong with conversion because with just periodLength it set time wrongly 🤔

Reply:
Now I'm confused as well :)

Please check to be sure, I might be wrong :)